machine/info.go

```diff
@@ -60,6 +61,16 @@ func Info(sysFs sysfs.SysFs, fsInfo fs.FsInfo, inHostNamespace bool) (*info.Mach
 		return nil, err
 	}
 
+	path, err := exec.LookPath("numactl")
```

Comment:
I'd prefer implementing the logic that is there in `numactl` instead of adding a dependency on this binary.